main/ui/src/main/java/org/cryptomator/ui/util/ListenerRegistry.java

```diff
@@ -0,0 +1,80 @@
+/*******************************************************************************
+ * Copyright (c) 2014 Sebastian Stenzel
```

Comment:
So should I put my name at the top of the notice and leave the list of contributors as is?